mp/src/vgui2/vgui_controls/ProgressBar.cpp

```diff
@@ -424,54 +425,50 @@ void ProgressBar::InitSettings()
 //-----------------------------------------------------------------------------
 void ProgressBar::OnDialogVariablesChanged(KeyValues *dialogVariables)
 {
-	if (!m_pszDialogVar.IsEmpty())
-	{
-		int val = dialogVariables->GetInt(m_pszDialogVar, -1);
-		if (val >= 0.0f)
-		{
-			SetProgress(val / 100.0f);
-		}
-	}
+    if (!m_pszDialogVar.IsEmpty())
+    {
+        int val = dialogVariables->GetInt(m_pszDialogVar, -1);
+        if (val >= 0.0f)
+        {
+            SetProgress(val / 100.0f);
+        }
+    }
 }
 
 
-DECLARE_BUILD_FACTORY( ContinuousProgressBar );
+DECLARE_BUILD_FACTORY(ContinuousProgressBar);
 
-//-----------------------------------------------------------------------------
-// Purpose: Constructor
-//-----------------------------------------------------------------------------
 ContinuousProgressBar::ContinuousProgressBar(Panel *parent, const char *panelName) : ProgressBar(parent, panelName)
 {
 }
 
-//-----------------------------------------------------------------------------
-// Purpose: 
-//-----------------------------------------------------------------------------
 void ContinuousProgressBar::Paint()
 {
-	int x = 0, y = 0;
-	int wide, tall;
-	GetSize(wide, tall);
+    int x = 0, y = 0;
+    int wide, tall;
+    GetSize(wide, tall);
 
-	surface()->DrawSetColor(GetFgColor());
+    surface()->DrawSetColor(GetFgColor());
 
-	switch( m_iProgressDirection )
-	{
-	case PROGRESS_EAST:
+    switch(m_iProgressDirection)
+    {
+    case PROGRESS_EAST:
     default:
-		surface()->DrawFilledRect( x, y, x + (int)( wide * _progress ), y + tall );
-		break;
-
-	case PROGRESS_WEST:
-		surface()->DrawFilledRect( x + (int)( wide * ( 1.0f - _progress ) ), y, x + wide, y + tall );
-		break;
-
-	case PROGRESS_NORTH:
-		surface()->DrawFilledRect( x, y + (int)( tall * ( 1.0f - _progress ) ), x + wide, y + tall );
-		break;
-
-	case PROGRESS_SOUTH:
-		surface()->DrawFilledRect( x, y, x + wide, y + (int)( tall * _progress ) );
-		break;
-	}
-}
\ No newline at end of file
+        surface()->DrawFilledRect(x, y, x + static_cast<int>(wide * _progress), y + tall);
+        break;
+
+    case PROGRESS_WEST:
+        surface()->DrawFilledRect(x + static_cast<int>(wide * (1.0f - _progress)), y, x + wide, y + tall);
+        break;
+
+    case PROGRESS_NORTH:
+        surface()->DrawFilledRect(x, y + static_cast<int>(tall * (1.0f - _progress)), x + wide, y + tall);
+        break;
+
+    case PROGRESS_SOUTH:
+        surface()->DrawFilledRect(x, y, x + wide, y + static_cast<int>(tall * _progress));
+        break;
+    }
+
+    PaintSubdivMarks(x, y, tall, wide);
```

Comment:
In both this function call and the other, x and y are always passed in as 0. Consider moving them to be function local, and making them to 0, removing the need to pass them in.